### DISCUSSION TITLE
Editable timestamp field in Keystonejs Admin UI

### DISCUSSION THREAD

**Discussion by jamesjing106** [keystonejs/keystone#7545] [Feature requests]
Last year, Mr. [BradNut] asked for a feature to edit the timestamp field in Keystone UI. It is time consuming to choose date via the keystone calendar UI. 

Is there any plan to let the timestamp UI editable? 

https://github.com/keystonejs/keystone/discussions/5307

**Comment by emmatown** [comment]
We don't support this feature at this time but it may be considered in the future. I have changed this question to a feature request in light of that.
